packages/typography/src/Link/Link.spec.tsx

```diff
@@ -155,4 +155,47 @@ describe('packages/typography', () => {
       expect(div!.tagName.toLowerCase()).toBe('div');
     });
   });
+
+  describe('anchor props', () => {
+    test('href is always passed to rendered component when the prop is supplied', () => {
+      renderLink({ href: 'test' });
+      const div = screen.getByText('Link').parentElement;
+      expect(div).toHaveAttribute('href');
+      expect(div?.getAttribute('href')).toBe('test');
+    });
+
+    test('it respects "target" and "rel" values when target is set', () => {
```

Comment:
Not sure if I like these test names since they imply they're both being set from target